app/views/questionnaire.py

```diff
@@ -241,16 +241,13 @@ def post_everyone_at_address_confirmation(eq_id, form_type, collection_id, group
 
 
 def validate_all_answers(answer_store, metadata):
```

Comment:
As this no longer validates all answers, I think we should rename this method something like `is_survey_completed`